subnet/etcd/local_manager.go

```diff
@@ -347,6 +347,10 @@ func (m *LocalManager) WatchLeases(ctx context.Context, cursor interface{}) (Lea
 	}
 }
 
+func (m *LocalManager) SetNodeNetworkUnavailableFalse(ctx context.Context) error {
```

Comment:
Can you add a comment on top of the function specifying that this function only makes sense for the kube subnet manager?

Reply:
look at new update